iop/cdvd/cdfs/src/imports.lst

```diff
@@ -0,0 +1,64 @@
+cdvdman_IMPORTS_start
+I_sceCdInit
+I_sceCdGetError
+I_sceCdRead
+I_sceCdStop
+I_sceCdSync
+I_sceCdDiskReady
+I_sceCdGetDiskType
+I_sceCdTrayReq
+I_sceCdSearchFile
+cdvdman_IMPORTS_end
+
+intrman_IMPORTS_start
+I_CpuSuspendIntr
+I_CpuResumeIntr
+intrman_IMPORTS_end
```

Comment:
I don't think these would be nessecary

Reply:
I have reviewed all the imports and I just have now the used ones